Automattic/wp-calypso
# Launchpad: Add the skip behavior for the Build intent

Related to:
* https://github.com/Automattic/wp-calypso/pull/80661
* D120528-code
* https://github.com/Automattic/jetpack/pull/32810

## Proposed Changes

* This PR adds the skip behavior for the Build intent. Skipping the fullscreen launchpad should redirect the users to Customer Home with the pre-launch launchpad. 
* After completing the Launch task, the post-launch Launchpad should be displayed

## Testing Instructions



- Apply the D120528-code to your sandbox
- Apply the https://github.com/Automattic/jetpack/pull/32810 to your sandbox. Since it was merged but not deployed yet you may need to run the following command:
```
bin/jetpack-downloader test jetpack-mu-wpcom-plugin trunk
```
- Access calypso.live
- Create a site with `/start`, use the `build` intent
![Screen Shot 2023-09-01 at 14 50 05](https://github.com/Automattic/wp-calypso/assets/1234758/c8e97365-3c3a-45e7-86b8-f88fb6a22238)
- Continue through the flow until you reach the fullscreen Launchpad
- Click on the `Skip for now` button
![Screen Shot 2023-09-01 at 14 52 46](https://github.com/Automattic/wp-calypso/assets/1234758/29386c7c-a8a2-4ccd-9b93-4c79c56c1c37)
- You should be redirected to the Customer Home with the pre-launch launchpad
- Click on the `Launch site` task
- You should see a celebration modal and then post-launch launchpad after you close the modal.


## Pre-merge Checklist
